engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-advanced-sync-options.html

```diff
@@ -0,0 +1,54 @@
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="standard" />
+      <span translate>
+        Optimized Sync
+      </span>
+    </label>
+    <div class="help-block">
+      <span translate>
+        Standard sync, optimized for speed by bypassing any unneeded steps.
+      </span>
+    </div>
+  </div>
+
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="skipMetadataCheck" />
+      <span translate>
+        Complete Sync
+      </span>
+    </label>
+    <div class="help-block">
+      <span translate>
+        Sync even if the upstream metadata appears to have no change and is only relevant for yum repositories.  This option will take longer than a normal sync.  Choose this option if:
+      </span>
+      <ul class="list-aligned">
+        <li translate>One or more RPMs are not showing up in the local repository even though they exist in the upstream repository.</li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="radio">
+    <label>
+      <input type="radio" ng-model="syncType" value="validateContents" />
+      <span translate>
+        Validate Content
+      </span>
+    </label>
+    <div class="help-block">
+      <span translate>
+        Performs a complete sync, verifying the checksum of all packages afterwards.  Any missing or corrupt RPMs will be re-downloaded.  This option will take the longest amount of time and is only relevant for yum repositories..  Choose this option if:
```

Comment:
Just to confirm, this option is the same as 'Complete Sync', but includes the checksum verification and additional file downloads (when mismatches occur).  Correct?